src/arsenic/services.py

```diff
@@ -91,11 +91,11 @@ async def _check_version(self):
                     "disable version checking, set `version_check` to "
                     "`False`."
                 )
-            version_str = match.group(1)
-            version = StrictVersion(version_str)
-            if version < StrictVersion("0.16.1"):
+            binary_version_str = match.group(1)
+            binary_version = version.parse(binary_version_str)
+            if binary_version < version.parse("0.16.1"):
```

Comment:
That version was release on `Apr 26, 2017`.

Perhaps this check can be removed entirely?

Reply:
tbh i'm fine with just removing the version check completely